### PR TITLE
Convert from BotFrameworkAdapter to CloudAdapter[bot-sso-adaptivecard…

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
…] Sample

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
After converting from BotFrameworkAdapter to CloudAdapter, review the testing results.
image

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/issues/1039

image

**Review by Prithvi-MSFT — APPROVED** [review]

File: samples/bot-sso-adaptivecard/nodejs/index.js [hunk 1]
```diff
@@ -3,34 +3,35 @@
 
 // index.js is used to setup and configure your bot
 
-// Import required packages
+// Import required pckages
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/bot-sso-adaptivecard/nodejs/index.js]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6965716924
image

**Comment by Pawank-MSFT** [comment]
Executed Build Pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7004027563

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Pull request merged**